File: PHP/program/Dpr.php
<?php

// Deklarasi Class
class Dpr {
    private $nama;
    private $id;
    private $bidang;
    private $partai;
    private $fotoProfil;

    // Konstruktor
    public function __construct($nama, $id, $bidang, $partai, $fotoProfil) {
        $this->nama = $nama;
        $this->id = $id;
        $this->bidang = $bidang;
        $this->partai = $partai;
        $this->fotoProfil = $fotoProfil;
    }

    // Getter dan Setter
    
    // Get nama
    public function getNama() {
        return $this->nama;
    }

    // Set nama
    public function setNama($nama) {
        $this->nama = $nama;
    }

    // Get id
    public function getId() {
        return $this->id;
    }

    // Set id
    public function setId($id) {
        $this->id = $id;
    }

    // Get bidang
    public function getBidang() {
        return $this->bidang;
    }

    // Set bidang
    public function setBidang($bidang) {
        $this->bidang = $bidang;
    }

    // Get partai
    public function getPartai() {
        return $this->partai;
    }

    // Set partai
    public function setPartai($partai) {
        $this->partai = $partai;
    }

    // Get foto profil
    public function getFotoProfil() {
        return $this->fotoProfil;
    }

    // Set pfoto profil
    public function setFotoProfil($fotoProfil) {
        $this->fotoProfil = $fotoProfil;
    }
}

File: Python/program/Dpr.py
# Deklarasi Class

class Dpr:

    # Privat (__), Protected (_), Public Atributes
    __name = ""
    __id = ""
    __bidang = ""
    __partai = ""

    # Konstruktor
    def __init__(self, name = "", id = "", bidang = "", partai = ""):
        self.__name = name
        self.__id = id
        self.__bidang = bidang
        self.__partai = partai

    # Getter dan Setter
        
    # Get Name
    def getName(self):
        return self.__name
    
    # Set Name
    def setName(self, name):
        self.__name = name

    # Get Id
    def getId(self):
        return self.__id
    
    # Set Id
    def setId(self, id):
        self.__id = id

    # Get Bidang
    def getBidang(self):
        return self.__bidang
    
    # Set Bidang
    def setBidang(self, bidang):
        self.__bidang = bidang

    # Get Partai
    def getPartai(self):
        return self.__partai
    
    # Set Partai
    def setPartai(self, partai):
        self.__partai = partai